CHANGES.md

```diff
@@ -71,6 +21,12 @@
 - cohttp-lwt: partly inline read_response, fix body stream leak (@madroach @dinosaure #696)
 - improve media type parsing (@seliopou #542, @dinosaure #725)
 - add comparison functions for Request.t and Response.t via ppx_compare (@msaffer-js @dinosaure #686)
+- [reverted] breaking changes to client and server API to use conduit
+  3.0.0 (@dinosaure #692). However, as the design discussion did not reach
+  consensus, these changes were reverted to preserve better compatibility with
+  existing cohttp users. (#741,  @samoht)
+
+## v3.0.0 - aborted
```

Comment:
Sorry to bother again.Can you rebase on master? I don’t understand why it is not showing up the changes from the marged PRs

Reply:
done